compiler/rustc_middle/src/ty/context.rs

```diff
@@ -2509,6 +2521,15 @@ impl<'tcx> TyCtxt<'tcx> {
         self.mk_substs(iter::once(self_ty.into()).chain(rest.iter().cloned()))
     }
 
+    pub fn mk_bound_variable_kinds<
+        I: InternAs<[ty::BoundVariableKind], &'tcx List<ty::BoundVariableKind>>,
+    >(
```

Comment:
```suggestion
    pub fn mk_bound_variable_kinds<I: InternAs<[ty::BoundVariableKind], &'tcx List<ty::BoundVariableKind>>,>(
```
`>(` artifact looks like a angry emote, nitpick is one might have a long line here instead

Reply:
This comes from rustfmt, but could just move this to a where clause